tests/functional/basic/test_simple_reference.py

```diff
@@ -168,21 +167,17 @@ def test_simple_reference(project):
     results = run_dbt()
     assert len(results) == 8
 
-    table_comp = TableComparison(
-        adapter=project.adapter, unique_schema=project.test_schema, database=project.database
-    )
-
     # Copies should match
-    table_comp.assert_tables_equal("users", "incremental_copy")
-    table_comp.assert_tables_equal("users", "materialized_copy")
-    table_comp.assert_tables_equal("users", "view_copy")
+    check_relations_equal(project.adapter, ["users", "incremental_copy"])
+    check_relations_equal(project.adapter, ["users", "materialized_copy"])
+    check_relations_equal(project.adapter, ["users", "view_copy"])
 
     # Summaries should match
-    table_comp.assert_tables_equal("summary_expected", "incremental_summary")
-    table_comp.assert_tables_equal("summary_expected", "materialized_summary")
-    table_comp.assert_tables_equal("summary_expected", "view_summary")
-    table_comp.assert_tables_equal("summary_expected", "ephemeral_summary")
-    table_comp.assert_tables_equal("summary_expected", "view_using_ref")
+    check_relations_equal(project.adapter, ["summary_expected", "incremental_summary"])
+    check_relations_equal(project.adapter, ["summary_expected", "materialized_summary"])
+    check_relations_equal(project.adapter, ["summary_expected", "view_summary"])
+    check_relations_equal(project.adapter, ["summary_expected", "ephemeral_summary"])
+    check_relations_equal(project.adapter, ["summary_expected", "view_using_ref"])
```

Comment:
Feels like we can merge all these checks into one check? Also okay to leave like this

Reply:
Good point. I've updated those tests. No point in leaving bad examples out there.